.claude/skills/rehearse-jobs/SKILL.md

```
@@ -0,0 +1,217 @@
+The script outputs NVRs like `kata-containers-3.25.0-4.rhaos4.19.el9`. Present the list
+to the user filtered by their selected OCP version(s) and let them pick.
+
+The `KATA_RPM_VERSION` is the NVR without the `kata-containers-` prefix, e.g. for
+`kata-containers-3.25.0-4.rhaos4.19.el9` the version is `3.25.0-4.rhaos4.19.el9`.
+
+If the user does **not** want to install a kata RPM, skip this step entirely.
+
+### Step 4: Edit the candidate configs
+
+The config files are at:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)